backend/tracim_backend/models/data.py

```diff
@@ -851,6 +851,8 @@ def has_new_information_for(self, user: User) -> bool:
 # TODO - G.M - 2018-06-177 - [author] Owner should be renamed "author"
 Index("idx__content_revisions__owner_id", ContentRevisionRO.owner_id)
 Index("idx__content_revisions__parent_id", ContentRevisionRO.parent_id)
+Index("idx__content_revisions__content_id", ContentRevisionRO.content_id)
+Index("idx__content_revisions__workspace_id", ContentRevisionRO.workspace_id)
```

Comment:
Please add a comment explaining that on mysql the index may have a different name

Reply:
done